### ISSUE TITLE
Auto-import files from bropages and tldr

### ISSUE THREAD

**Issue by cweiske** [JIghtuse/manpages-tldr#5]
It would be nice to have all the contents from tldr and bropages in manpages-tldr, with a way to update them from their repositories.

**Comment by JIghtuse** [comment]
@cweiske Yeah, I just thought about that. `pandoc` does a great job converting Markdown files to manpages, but we need to add some hints (like `# NAME` and `#EXAMPLES`) to get better looking pages.

**Comment by JIghtuse** [comment]
- [x] script to import tldr pages
- [ ] script to import bropages

**Comment by JIghtuse** [comment]
So, I've made a script to import tldr pages :smile_cat: lots of new pages!

**Comment by cweiske** [comment]
What about license compatibility? Are the manpages-tldr license and the tldr pages license compatible?

**Comment by JIghtuse** [comment]
@cweiske yes, this project initially kinda based on tldr-pages and I took the same license (MIT).

**Comment by cweiske** [comment]
I think tldr import is enough. Once tldr is done with their bropages support (https://github.com/tldr-pages/tldr/issues/266), the current tldr script would cover everything automatically.

**Comment by JIghtuse** [comment]
@cweiske oh, nice! Would waiting for this import then.
